Automattic/jetpack
# Update/blocks seo tools discoverability

Fixes #33161

## Proposed changes:

- Always register the Seo Tools editor extension.
- Add a new placeholder when the panel is originally loading.
- Add a placeholder with a button to enable the module when it is disabled.

### Other information:



## Does this pull request change what data or activity we track or use?

* Yes: views of the placeholder (jetpack_editor_seo_enable)
## Testing instructions:

* Spin this branch locally, for example, connect via JT
* Go to Posts > Add New
* Click on the Jetpack plugin sidebar.
* Click on the "SEO" panel title.
* Click on the button.
* Verify that the module is enabled and working as expected
